composer/utils/object_store/oci_object_store.py

```diff
@@ -16,6 +16,26 @@
 __all__ = ['OCIObjectStore']
 
 
+def _reraise_oci_errors(uri: str, e: Exception):
+    import oci
+    # If it's an oci service error with code: ObjectNotFound or status 404
+    if isinstance(e, oci.exceptions.ServiceError):
+        if e.status == 404: # type: ignore
+            if e.code == 'ObjectNotFound':  # type: ignore
+                raise FileNotFoundError(f'Object {uri} not found. {e.message}') from e
```

Comment:
```suggestion
            if e.code == 'ObjectNotFound':  # type: ignore
                raise FileNotFoundError(f'Object {uri} not found.') from e
```
The message should be shown because you're doing from e

Reply:
Actually, for oci.exceptions.service_errors it's a lot prettier to extract the message because if you raise the error it prints a lot of other stuff. See [this code](https://github.com/oracle/oci-python-sdk/blob/master/src/oci/exceptions.py#L64). They actually pass the error_details to the Exception constructor not the message, so it's a lot more verbose if you raise the error without printing the message first